src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -577,6 +579,13 @@ public T Attach<T>(JsonRpcProxyOptions options)
             return proxy;
         }
 
+        public void Relay(Stream relaySendingStream, Stream relayReceivingStream, object relayTarget = null)
+        {
+            this.relayRpc = JsonRpc.Attach(relaySendingStream, relayReceivingStream, relayTarget);
+
+            this.relayRpc.relayRpc = this;
```

Comment:
This appears to cause a circular loop between the two `JsonRpc` instances in which the original one is imperceptible from the one created here. Is that right (and good)?